contributors/guide/pull-requests.md

```diff
@@ -72,7 +72,7 @@ Here's the process the pull request goes through on its way from submission to m
 
 1. If you're **not** a member of the Kubernetes organization, a Reviewer/Kubernetes Member checks that the pull request is safe to test. If so, they comment `/ok-to-test`. Pull requests by Kubernetes organization [members](/community-membership.md) do not need this step. Now the pull request is considered to be trusted, and the pre-submit tests will run:
 
-    1. Automatic tests run. See the current list of tests on the [MERGE REQUIREMENTS tab, at this link](https://submit-queue.k8s.io/#/info)
+    1. Automatic tests run. See the current list of tests on the [MERGE REQUIREMENTS tab, at this link](https://prow.k8s.io/tide#/info)
```

Comment:
This link doesn't actually show tests that are run for every PR.  We're sort of lacking a clear list:
- The presubmits themselves are spread amongst a bunch of yaml files in https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes
- We could point to https://k8s-testgrid.appspot.com/presubmits-kubernetes-blocking
- We could point to https://prow.k8s.io/?repo=kubernetes%2Fkubernetes&type=presubmit

I guess the last link is my vote. But none of these clearly answer "how can I find out what these jobs do"